nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -590,28 +619,59 @@ private void configureConnectors(final Server server) throws ServerConfiguration
 
             logger.info("Configuring Jetty for HTTPs on port: " + port);
 
-            // add some secure config
-            final HttpConfiguration httpsConfiguration = new HttpConfiguration(httpConfiguration);
-            httpsConfiguration.setSecureScheme("https");
-            httpsConfiguration.setSecurePort(props.getSslPort());
-            httpsConfiguration.addCustomizer(new SecureRequestCustomizer());
+            final List<Connector> serverConnectors = Lists.newArrayList();
 
-            // build the connector
-            final ServerConnector https = new ServerConnector(server,
-                    new SslConnectionFactory(createSslContextFactory(), "http/1.1"),
-                    new HttpConnectionFactory(httpsConfiguration));
+            final Map<String, String> httpsNetworkInterfaces = props.getHttpsNetworkInterfaces();
+            if (httpsNetworkInterfaces.isEmpty()) {
```

Comment:
The following comment from line 578 applies here as well. [https://github.com/apache/nifi/pull/1508/commits/88520369d910dff61892b86841dc9eab28bf03e3#r101147915](url)